### PR TITLE
fix(benchmarks): update react and polymer benchmarks and get tree upd…

### PR DESCRIPTION
**Pull request by tbosch**
…ate numbers for all of the benchmarks as well.

### CODE REVIEW

File: modules/benchmarks_external/e2e_test/polymer_tree_perf.ts [hunk 1]
```diff
@@ -6,11 +6,21 @@ describe('polymer tree benchmark', function() {
...
+      id: 'polymer.tree.create',
+      params: [{name: 'depth', value: 9, scale: 'log2'}],
+      waitForAngular2: false
+    }).then(done, done.fail);
+  });
+
+  it('should log the stats (update)', function(done) {
+    runClickBenchmark({
+      url: URL,
+      buttons: ['#createDom'],
```
Comment (yjbanov) [inline modules/benchmarks_external/e2e_test/polymer_tree_perf.ts]: is this the correct button?

File: modules/benchmarks_external/e2e_test/static_tree_perf.ts [hunk 2]
```diff
@@ -6,11 +6,21 @@ describe('ng1.x tree benchmark', function() {
...
+      id: 'ng1.static.tree.create',
+      params: [],
+      waitForAngular2: false
+    }).then(done, done.fail);
+  });
+
+  it('should log the stats (update)', function(done) {
+    runClickBenchmark({
+      url: URL,
+      buttons: ['#createDom'],
```
Comment (yjbanov) [inline modules/benchmarks_external/e2e_test/static_tree_perf.ts]: same here

File: modules/benchmarks_external/e2e_test/tree_perf.ts [hunk 3]
```diff
@@ -6,11 +6,21 @@ describe('ng1.x tree benchmark', function() {
...
+      id: 'ng1.tree.create',
+      params: [{name: 'depth', value: 9, scale: 'log2'}],
+      waitForAngular2: false
+    }).then(done, done.fail);
+  });
+
+  it('should log the stats (update)', function(done) {
+    runClickBenchmark({
+      url: URL,
+      buttons: ['#createDom'],
```
Comment (yjbanov) [inline modules/benchmarks_external/e2e_test/tree_perf.ts]: and here

**Comment by yjbanov** [comment]
If you think you're pressing the right buttons, then LGTM&merge.

**Comment by tbosch** [comment]
Closes https://github.com/angular/angular/issues/960

**Comment by tbosch** [comment]
Blocked by the router not working in Angular 1.4

File: npm-shrinkwrap.json [hunk 4]
```diff
@@ -8,9 +8,9 @@
...
     "angular": {
-      "version": "1.3.5",
-      "from": "https://registry.npmjs.org/angular/-/angular-1.3.5.tgz",
-      "resolved": "https://registry.npmjs.org/angular/-/angular-1.3.5.tgz"
+      "version": "1.4.7",
+      "from": "angular@1.4.7",
+      "resolved": "https://registry.npmjs.org/angular/-/angular-1.4.7.tgz"
     },
     "angular-animate": {
       "version": "1.3.5",
```
Comment (btford) [inline npm-shrinkwrap.json]: Looks like you forgot to bump the other `angular-*` packages, which is why the router tests fail.

File: package.json [hunk 5]
```diff
@@ -33,7 +33,7 @@
     "zone.js": "0.5.8"
   },
   "devDependencies": {
-    "angular": "1.3.5",
+    "angular": "^1.4.7",
     "angular-animate": "1.3.5",
```
Comment (btford) [inline package.json]: You need to bump angular-animate and angular-mocks below as well.

**Pull request closed without merging**